### ISSUE TITLE
Question regarding InversifyJS documentation

### ISSUE THREAD

**Issue by vesteraas** [inversify/InversifyJS#281]
Just wanted to point out the following code, which can be found in **The Basics (TypeScript), step 2**:

```
import { injectable, inject } from "inversify";
import "reflect-metadata";

@injectable()
class Katana implements Katana {
    public hit() {
        return "cut!";
    }
}

@injectable()
class Shuriken implements Shuriken {
    public throw() {
        return "hit!";
    }
}
```

Here, the classes implements interfaces with the same name.  I think this is confusing.  Could you please provide a working example using interfaces?

Sorry for not using the issue template.

**Comment by remojansen** [comment]
Thanks for reporting :) This will be fixed for the 2.0.0 release.

**Comment by remojansen** [comment]
This was done https://github.com/inversify/InversifyJS/pull/282
